odoo/models.py

```diff
@@ -1044,7 +1044,7 @@ def export_data(self, fields_to_export):
         return {'datas': self._export_rows(fields_to_export)}
 
     @api.model
-    def load(self, fields, data):
+    def load(self, fields, data, key_fields=None):
```

Comment:
In earlier commits I used the ORM instead - but after considering performance I thought SQL would be faster.
I have added a check to verify that these keys are allowed.
After my last changes, I'm not sure how it could be abused (password example) any more than the regular import?